packages/itwin/geo-tools/src/GeoToolsUiItemsProvider.tsx

```diff
@@ -2,28 +2,20 @@
 * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
-import type {
-  CommonToolbarItem,
-  UiItemsProvider,
-} from "@itwin/appui-abstract";
-import {
-  StageUsage,
-  ToolbarOrientation,
-  ToolbarUsage,
-} from "@itwin/appui-abstract";
-import { ToolbarHelper } from "@itwin/appui-react";
+// eslint-disable-next-line @typescript-eslint/consistent-type-imports
```

Comment:
why is this disabled? can just run `lint --fix`

Reply:
The fix normally be to use:
`import type UiItemsProvider from "@itwin/appui-react";`

When I use this, I get a compile error:
`src/GeoToolsUiItemsProvider.tsx(10,55): error TS2709: Cannot use namespace 'UiItemsProvider' as a type.`

Reply:
> The fix normally be to use: `import type UiItemsProvider from "@itwin/appui-react";`
> 
> When I use this, I get a compile error: `src/GeoToolsUiItemsProvider.tsx(10,55): error TS2709: Cannot use namespace 'UiItemsProvider' as a type.`

import { type UiItemsProvider } from "@itwin/appui-react" / import type { UiItemsProvider } from "@itwin/appui-react"

Reply:
Thanks Jason, I was missing only the curly brackets... the lint message could be improved here.